src/rpc/registry/methods_reg.rs

```diff
@@ -0,0 +1,293 @@
+// Copyright 2019-2025 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::lotus_json::HasLotusJson;
+use crate::shim::machine::BuiltinActor;
+use crate::shim::message::MethodNum;
+use ahash::{HashMap, HashMapExt};
+use anyhow::{Result, anyhow};
+use cid::Cid;
+use once_cell::sync::Lazy;
+use serde::de::DeserializeOwned;
+use serde_json::Value;
+
+use crate::rpc::registry::actors_reg::{ACTOR_REGISTRY, get_actor_type_from_code};
+
+// Global registry for method parameter deserialization
+static METHOD_REGISTRY: Lazy<MethodRegistry> = Lazy::new(|| {
+    let mut registry = MethodRegistry::new();
+    register_known_methods(&mut registry);
+    registry
```

Comment:
Would it make sense to make `register_known_methods` an associated function (constructor) of the `MethodRegistry`?

Reply:
Done